bundle.Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM scratch
 LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
-LABEL operators.operatorframework.io.bundle.package.v1=cockroachdb-certified
+LABEL operators.operatorframework.io.bundle.package.v1=cockroach-operator
```

Comment:
Nothing to worry about here, this file is auto-generated but not used. We could gitignore it if that's preferred.